bucket/miniconda3.json

```
@@ -46,7 +46,7 @@
     ],
     "persist": "envs",
     "checkver": {
-        "url": "https://docs.anaconda.com/free/miniconda/miniconda-other-installer-links/",
+        "url": "https://repo.anaconda.com/miniconda/",
         "regex": "Miniconda3-py(?<py>\\d+)_([\\d.-]+)-Windows"
```

```suggestion
        "regex": "Miniconda3-py(?<py>\\d{3})_([\\d.-]+)-Windows"
```
Otherwise it will extract `py39` before `py312`, please double check.